src/panels/logbook/ha-logbook.ts

```diff
@@ -48,7 +50,7 @@ class HaLogbook extends LitElement {
   protected render(): TemplateResult {
     if (!this.entries?.length) {
       return html`
-        <div class="container">
+        <div class="container" .dir=${emitRTLDirection(this._rtl)}>
```

Comment:
You can remove the ignore on line 31 now

Reply:
Done